Wikipedia/assets/index.js

```diff
@@ -16,13 +16,6 @@ function stringEndsWith(str, suffix) {
     return str.indexOf(suffix, str.length - suffix.length) !== -1;
 }
 
-function getZoomLevel() {
```

Comment:
@coreyfloyd this buggy method was causing a TOC bug in @joewalsh's iPad article refactor.